### PR TITLE
Modifying vcf_to_plink

### PR DESCRIPTION
**Pull request by slzarate**
README and dxapp.json updates

### CODE REVIEW

File: sequencing/vcf_to_plink/Readme.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
-# vcf_to_plink (DNAnexus Platform App)
+# Convert VCF file to PLINK file (DNAnexus Platform App)
 
-Uses PLINK 1.9 to convert VCF files to PLINK bed/bim/fam files.
```
Comment (yihchii) [inline sequencing/vcf_to_plink/Readme.md]: "What does this applet do?" -> "What does this app do?"
I am assuming that all applets will eventually be published as apps, for the paper's publication

**Review by jdidion — APPROVED** [review]

File: sequencing/vcf_to_plink/Readme.md [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
+This app uses PLINK 1.9 to convert VCF files (`*.vcf.gz`) to PLINK bed/bim/fam files (`*.bed`, `*.bed`, `*.fam`).
 
 ## What data are required for this app to run?
 
-## Usage
-### Options
+This app requires an equal number of VCF files (`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`).
 
-## What does this app output?
+## What does this applet output?
```
Comment (yihchii) [inline sequencing/vcf_to_plink/Readme.md]: What does this applet output?" -> "What does this app output?"
I am assuming that all applets will eventually be published as apps, for the paper's publication

File: sequencing/vcf_to_plink/dxapp.json [hunk 3]
```diff
@@ -146,7 +146,5 @@
     "whatsNew": "*1.0.0: Cleaned up, added documentation, moved to Ubuntu 14.04;",
```
Comment (yihchii) [inline sequencing/vcf_to_plink/dxapp.json]: "whatsNew": "*1.0.0: Cleaned up, added documentation, moved to Ubuntu 14.04;", 
->
"whatsNew": "* 1.0.0 First version of the Convert VCF file to PLINK file app."

For generalized first app release log.

File: sequencing/vcf_to_plink/dxapp.json [hunk 4]
```diff
@@ -10,7 +10,7 @@
   ],
   "inputSpec": [
     {
-      "name": "vcf_fn",
+      "name": "variants_vcfgz",
```
Comment (yihchii) [inline sequencing/vcf_to_plink/dxapp.json]: Should this name end with an s for "array" of files?
"variants_vcfgz" -> "variants_vcfgzs"
I'll leave this for you to make a final decision here.

File: sequencing/vcf_to_plink/dxapp.json [hunk 5]
```diff
@@ -20,7 +20,7 @@
       ]
     },
     {
-      "name": "vcfidx_fn",
+      "name": "variants_vcfgztbi",
```
Comment (yihchii) [inline sequencing/vcf_to_plink/dxapp.json]: Should this name end with an s for "array" of files?
"variants_vcfgztbi" -> "variants_vcfgztbis"
I'll leave this for you to make a final decision here.

**Review by yihchii — COMMENTED** [review]
LGTM!

**Pull request merged**